### DISCUSSION TITLE
LPCMCI implementation

### DISCUSSION THREAD

**Discussion by jjakenichol** [jakobrunge/tigramite#146] [Ideas]
Hi, will the recent LPCMCI algorithm be implemented? I ran across the abstract from a talk you all gave this year: https://meetingorganizer.copernicus.org/EGU21/EGU21-8259.html. I'm interested in open, natural systems so this idea is very appealing to me. Is there a paper out on the algorithm yet? I'd love to read more, specifically how it can forego the sufficiency assumption and what advantages it has over others that do that, like FCI.

Thanks!

**Comment by jakobrunge** [comment]
Hi, it is currently available directly in the folder neurips2020. It is fully usable there. 

We will integrate it into the main package soon. (Then it is available directly as a function in PCMCI and also in pip).

The accompanying paper with lots of comparison studies is here: https://proceedings.neurips.cc/paper/2020/hash/94e70705efae423efda1088614128d0b-Abstract.html

**Reply by jjakenichol** [reply]
Perfect, thank you.
